plugins/mcreator-localization/lang/texts.properties

```diff
@@ -481,7 +484,7 @@ workspace.elements.list.filter_all=Show all
 workspace.elements.list.filter_locked=Locked elements
 workspace.elements.list.filter_witherrors=Elements with build errors
 workspace.elements.list.ascending=Ascending
-workspace.category.mod_elements=Mod  elements
```

Comment:
This is intentional for UI layout so please revert this and don't do unrelated changes in PRs

Reply:
fixed